### DISCUSSION TITLE
Mark an area

### DISCUSSION THREAD

**Discussion by Kronzky** [devemux86/cruiser#603] [Q&A] [answered]
Is there a way that I could highlight or mark a certain area on the map?
I will be moving _within_ that area (_without_ any routing), and need some indicator of where the boundaries are.
![markedarea](https://github.com/user-attachments/assets/606e57fb-ca4f-4b9f-a9b3-46005a76222d)

Is it possible to create a customized map, and use that instead? Or what other methods might work?
I guess the upcoming no-go zones might be a way to use them in the "opposite" way?

**Comment by devemux86** [comment]
@Kronzky A solution is to create a route with beelines, store it in favorites and display it as overlay.

---

Cruiser 5 will allow importing all kinds of shapes from many formats and saving them in favorites.
Then they can be used as overlays or no-go areas.

Also draw no-go areas as circles that is the simplest area shape.

**Reply by Kronzky** [reply]
What do you mean by "beelines"? 
I tried drawing straight lines in BRouter, but when I then import that route into Cruise I often get an "impossible route" error.

No-go circles are unfortunately too coarse to specify an area accurately enough, but I guess I'll have to wait for V5 to use more varied shapes.

**Reply by devemux86** [reply]
@Kronzky 

> What do you mean by "beelines"?

Select "Straight line" when planning a route with BRouter.
Straight lines do not require route calculation in Cruiser.

Create a route, change its profile to "Straight line" and use the "Extend route" menu to create any shape with straight lines.

>  I tried drawing straight lines in BRouter, but when I then import that route into Cruise I often get an "impossible route" error.

To maintain all route data, must export / import the route in GPX (but select navigation) or JSON / Kurviger format.

**Reply by Kronzky** [reply, marked as answer]
I got it now!
Creating a straight line route in BRouter, and then importing it as an Overlay will give me a nice border that I can make as detailed as I need it.

**Reply by devemux86** [reply]
We could probably offer a design tool in Cruiser for lines and polygons,
that can be stored in the new "My collection" that supports all shape types.

Then they can be used as overlays or no-go areas.

**Comment by devemux86** [comment]
> I will be moving within that area (without any routing), and need some indicator of where the boundaries are.

@Kronzky Do you also suggest a new feature in navigation / follow?

Introduce alarms when we enter some marked areas in the map?

**Reply by Kronzky** [reply]
No, I think that would be overkill. After all, if I can see my area borders when I have the map open that'd be good enough.
It's probably too unusual a use-case for it to warrant a dedicated feature.
More fine-grained no-go zones will probably do the job just fine.

**Comment by GpxFeed** [comment]
With "GpsVisualizer" you can draw a polygon online. In the drawing tools, you have change "trk" to "area" by clicking on it - a bit tricky to get through.

https://www.gpsvisualizer.com/draw/

One offline tool I know of is RouteConverter.

I have already used such polygons for a script to punch out pois with a polygon like with a biscuit cutter.
